backend/indexer.py

```diff
@@ -604,7 +604,10 @@ def index_meta(self):
 
 
 if __name__ == "__main__":
-    redis_conn = redis.from_url(os.environ.get("REDIS_URL", "redis://localhost:6379"))
+    redis_conn = redis.from_url(
+        os.environ.get("REDIS_URL", "redis://localhost:6379"),
+        decode_responses=True
+    )
```

Comment:
We should probably have a util `get_redis_connection()` or even `utils.redis_connection`.

Reply:
This is a very good idea that I don't know why I didn't think of it. I've added a `backend.util.redis_connection` in 1525410.